staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -693,6 +697,7 @@ function kube::codegen::gen_client() {
 function kube::codegen::gen_register() {
     local in_dir=""
     local boilerplate="${KUBE_CODEGEN_ROOT}/hack/boilerplate.go.txt"
+    local version="latest"
```

Comment:
Remove this - not used

Reply:
Fixed in 4f4bedd0888034127e6d309ae3ede2e1e2cae9c5